### ISSUE TITLE
Fuze is gone

### ISSUE THREAD

**Issue by Nicolas62x** [Fuzay/ChaosGuardianKiller#35]

**Comment by solstice-r** [comment]
Maybe you should just fork this and continue on the fork, and if fuze comes back, merge it with this repo?

**Comment by Nicolas62x** [comment]
I don’t think he will come back

**Comment by BlackNiteHD** [comment]
Yes i doesn't comeback because it is a french youtuber and he finish is modded series so i think that can be finish right now but later when he wants !

**Comment by dadodasyra** [comment]
@Fuzay you spoke about the project again in one of your last videos. Do you think you will take it back? I think @Nicolas62x  would be happy to comeback with the new skills he has learned.

**Comment by Nicolas62x** [comment]
> @Fuzay you spoke about the project again in one of your last videos. Do you think you will take it back? I think @Nicolas62x would be happy to comeback with the new skills he has learned.

Lol

**Comment by omerien** [comment]
FuzeIII started this project in a season of "Minecraft Moddé", he gone to another season and now he stopped "Minecraft Moddé". We can actually say it's a dead project.

**Comment by solstice-r** [comment]
Wait he entirely stopped playing modded Minecraft? Sent from Mail for Windows 10 From: omerienSent: Saturday, 19 June 2021 11:52To: Fuzay/ChaosGuardianKillerCc: Nath_SoM; CommentSubject: Re: [Fuzay/ChaosGuardianKiller] Fuze is gone (#35) FuzeIII started this project in a season of "Minecraft Moddé", he gone to another season and now he stopped "Minecraft Moddé". We can actually say it's a dead project.—You are receiving this because you commented.Reply to this email directly, view it on GitHub, or unsubscribe.

**Comment by omerien** [comment]
He stopped because he hadn't more pleasure on it. There is suspense on a new season but for the moment, yes he stopped.
Edit : he still plays Modded Minecraft but the series of videos where he was doing this stuff are stopped for the moment.

**Comment by dadodasyra** [comment]
Yep so he announced yesterday or the day before that he will resume with a lot more role play, the project not being very R.P. I don't think he will resume it

**Comment by omerien** [comment]
Bon par contre on peut arrêter de parler anglais je regarde et tout le monde dans la discussion est fr
